autofill/autoprompt/case-1.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Autofill autoprompt for signin forms</title>
+  <link rel="stylesheet" href="../style.css" />
+</head>
+
+<body>
+<p><a href="../index.html">[Home]</a></p>
```

Comment:
this doesn't work because we are two levels deep (autofill/autoprompt/). Same for all test case pages.